### PR TITLE
Avoid frequent changes of a bug assignee

### PR DESCRIPTION
**Pull request by suhaibmujahid**
This should mitigate cases like in [bug 1575465](https://bugzilla.mozilla.org/show_bug.cgi?id=1575465).

This PR is based on a discussion with @calixteman.

### CODE REVIEW

**Comment by suhaibmujahid** [comment]
The dry-run querys: 
- [Before the PR](https://bugzilla.mozilla.org/buglist.cgi?include_fields=assigned_to&include_fields=triage_owner&include_fields=flags&include_fields=id&include_fields=summary&include_fields=groups&include_fields=product&include_fields=component&resolution=---&f1=assignee_last_login&o1=lessthan&v1=2021-08-26%2000%3A00%3A00%2B00%3A00&f2=days_elapsed&o2=lessthan&v2=720&j4=OR&f4=OP&f5=assigned_to&o5=equals&v5=nobody%40mozilla.org&f6=assigned_to&o6=regexp&v6=^.*\.bugs%24&f7=assigned_to&o7=isempty&f8=CP&n4=1&f9=status_whiteboard&o9=notsubstring&v9=[no-nag]&product=Core&product=DevTools&product=External%20Software%20Affecting%20Firefox&product=Fenix&product=Firefox&product=Firefox%20for%20iOS&product=Firefox%20Build%20System&product=GeckoView&product=JSS&product=NSPR&product=NSS&product=Remote%20Protocol&product=Testing&product=Toolkit&product=Web%20Compatibility&product=WebExtensions&list_id=16034317)
- [After the PR](https://bugzilla.mozilla.org/buglist.cgi?include_fields=assigned_to&include_fields=triage_owner&include_fields=flags&include_fields=id&include_fields=summary&include_fields=groups&include_fields=product&include_fields=component&resolution=---&f1=assignee_last_login&o1=lessthan&v1=2021-08-26%2000%3A00%3A00%2B00%3A00&f2=days_elapsed&o2=lessthan&v2=720&n3=1&f3=assigned_to&o3=changedafter&v3=-24m&j4=OR&f4=OP&f5=assigned_to&o5=equals&v5=nobody%40mozilla.org&f6=assigned_to&o6=regexp&v6=%5E.%2A%5C.bugs%24&f7=assigned_to&o7=isempty&f8=CP&n4=1&f9=status_whiteboard&o9=notsubstring&v9=%5Bno-nag%5D&product=Core&product=DevTools&product=External%20Software%20Affecting%20Firefox&product=Fenix&product=Firefox&product=Firefox%20for%20iOS&product=Firefox%20Build%20System&product=GeckoView&product=JSS&product=NSPR&product=NSS&product=Remote%20Protocol&product=Testing&product=Toolkit&product=Web%20Compatibility&product=WebExtensions&list_id=16034317)

This PR reuses the threshold `last_activity_months` to exclude bugs that their assignees change recently, currently, the threshold value is 24 months. It could be worth using a separate threshold.

**Review by marco-c — CHANGES_REQUESTED** [review]
The threshold is too large, this way we won't notice bugs with inactive assignees if they have been assigned to the bug within the past two years.
We should use something very short, like a week or two.

**Comment by suhaibmujahid** [comment]
> The threshold is too large, this way we won't notice bugs with inactive assignees if they have been assigned to the bug within the past two years. We should use something very short, like a week or two.

I agree, dee597db83192231a0f7407a69699451abc12a03 sets an independent threshold with a default value of 2 weeks.

File: auto_nag/scripts/configs/tools.json [hunk 1]
```diff
@@ -125,6 +125,7 @@
     "number_of_comments": 2
   },
   "assignee_no_login": {
+    "unasign_weeks": 2,
```
Comment (marco-c) [inline auto_nag/scripts/configs/tools.json:128]:
```suggestion
    "unassign_weeks": 2,
```

**Review by marco-c — APPROVED** [review]
Thanks!

**Pull request merged**